library/logicjavascript.js

```diff
@@ -29156,4 +29156,26 @@ function solveFraction(num1, num2) {
         result += `\\[\\frac{${tempnum1}}{${tempnum2}} = ${sol}\\]`;
     }
     return { 'result': result, 'steps': temp };
-}
\ No newline at end of file
+}
+function interquartile(q1,q3)
+{
+    var ans="";
+    var iq="";
+    q1=parseFloat(q1);
+    ans += "Step 1: Input the lower quartile range => " + q1;
+    var q3=parseFloat(q3);
+    ans += "Step 2: Input the upper quartile range => " + q3;
+
+    if(q1=="" || q2=="")
+    { 
+    ans+="Enter all values";}
+    else
+    {
+        ans+="Step3: Calculate the inter quartile range";
+        
+        iq=q3-q1;
+        ans+="The Interquartile range of"+$q3$-$q1$;
+        ans+=iq;   
+    }
+    return{'result':iq,'steps:':ans};
```

Comment:
you wrote 'steps:' : inside the double quotes, use 'steps' not 'steps:'

Reply:
> you wrote 'steps:' : inside the double quotes, use 'steps' not 'steps:'

done.